db/migrate/20190722123136_add_initial_password_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddInitialPasswordToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :initial_password, :string
```

Comment:
no need of this new column 
in your case when you create new user then update its password with random generated value